.changelog/3003.feature.md

```diff
@@ -0,0 +1,4 @@
+go/oasis-node/cmd/stake: Add `account get_address` commmand
+
+Add `oasis-node stake account get_address` CLI command for obtaining a
```

Comment:
Feedback on the (sub)comand's name (or position in the hierarchy) are welcome.

Reply:
Maybe call it `pubkey2address` (or `pk2addr`) or something to be more obvious what it does? Having it under stake/account seems fine.

Reply:
> Maybe call it `pubkey2address` (or `pk2addr`) or something to be more obvious what it does?

Agreed. I've changed it to `pubkey2address`.

> Having it under stake/account seems fine.

On a second though, I think putting it directly under `oasis-node stake` makes more sense.